lib/config/guiconfiguration.go

```diff
@@ -36,12 +36,12 @@ func (c GUIConfiguration) Address() string {
 		if strings.Contains(override, "/") {
 			url, err := url.Parse(override)
 			if err != nil {
-				return override
+				return []string{override}
 			}
-			return url.Host
+			return []string{url.Host}
 		}
 
-		return override
+		return []string{override}
```

Comment:
Should probably add support for comma notation for multiple addresses.

Reply:
I am not sure to see your point at the level of this function. Where would you precisely introduce support for comma notation?

Reply:
STGUIADDRESS env var override should probably support comma notation, as in `STGUIADDRESS=http://0.0.0.0:80,https://0.0.0.0:443`

Reply:
Fine.

Reply:
Here is my proposal to handle environment overriding with multiple listeners:
1. First we check for a new environment variable `STGUIADDRESSES` which supports a comma separated list of individual urls;
2. If the first one is not defined, we fallback to the legacy `STGUIADDRESS` environment variable which only support one url;
3. If the second one is not defined, we use the listeners as defined in the configuration.

Reply:
Sounds good to me. @calmh?

Reply:
sgtm